testing/features/navigation.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Navigation component
+
+  The greedy nav displays as many navigation links as fit on the viewport. 
+  Any that don't fit are added into a expandable dropdown.
+
+  Background:
+    Given the navigation is on the page
+
+    @not_mobile
+    Scenario: all links fit onto the screen
+        Then the dropdown toggle is not present
+
+    Scenario: screen is too narrow to show all the links
+        Given the navigation does not fit on the screen
+        Then the dropdown toggle is present
+        And the dropdown menu is hidden
+
+    Scenario: clicking the more button shows the dropdown menu
+        Given the navigation does not fit on the screen
+        When I click the More button
+        Then the dropdown menu is present
+
+    Scenario: clicking the close button hides the dropdown menu
+        Given the dropdown menu is already open 
+        When I click the Close button
+        Then the dropdown menu is hidden
+
+    Scenario: tabbing onto the dropdown toggle opens the dropdown menu
+        Given the navigation does not fit on the screen
+        And the dropdown toggle is present
+        When I tab onto the More button 
+        Then the Close button is present
+        And the dropdown menu is present
+
+    @failing
```

Comment:
This currently works on live because of the presence of header links

Reply:
Rightio. Is this something we have a ticket for. We could add it to this as an `@` tag like `@NP-1244`